nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteStreamCommand.java

```diff
@@ -419,21 +422,27 @@ public void onTrigger(ProcessContext context, final ProcessSession session) thro
                 }
                 return 0;
             });
-            for ( final PropertyDescriptor descriptor : propertyDescriptors) {
+
+            for (final PropertyDescriptor descriptor : propertyDescriptors) {
+                if (descriptor.isSensitive()) {
+                    String maskedValue = IntStream.rangeClosed(0, MASKED_VALUE_LENGTH).mapToObj(i -> "*").collect(Collectors.joining());
+                    argumentAttributeValue.add(maskedValue);
```

Comment:
Instead of the `IntStream` approach, this should be replaced with a static variable named something like `MASKED_ARGUMENT` with a value of `********`. That would allow simplifying this as follows:

```suggestion
                    argumentAttributeValue.add(MASKED_ARGUMENT);
```

Reply:
okay i will change